jax/experimental/jax2tf/tests/sharding_test.py

```diff
@@ -28,26 +28,24 @@
 import unittest
 
 from absl.testing import absltest
+import numpy as np
+import tensorflow as tf  # type: ignore[import]
```

Comment:
Would importing tensorflow before JAX affect something?

cc @hawkinsp

Reply:
I don't think it will, but I reverted anyway.